core-it-suite/src/test/java/org/apache/maven/it/MavenITmng4470AuthenticatedDeploymentToProxyTest.java

```diff
@@ -92,7 +89,7 @@ public void handle( String target, Request baseRequest, HttpServletRequest reque
                 if ( auth != null )
                 {
                     auth = auth.substring( auth.indexOf( ' ' ) + 1 ).trim();
-                    auth = B64Code.decode( auth, StandardCharsets.US_ASCII.name() );
+                    auth = new String( java.util.Base64.getDecoder().decode( auth ), StandardCharsets.US_ASCII );
```

Comment:
Why not `import java.util.Base64`?

Reply:
no special reason, I believe at one point both classes were present (Jetty deprecated B64code and j.u.Base64) and my IDE just choose to fully name it? Should I add it to import?

Reply:
😄